modules/serverless-deprecated-removed-features.adoc

```diff
@@ -13,14 +13,39 @@ For the most recent list of major functionality deprecated and removed within {S
 // OCP + OSD table
 
 .Deprecated and removed features tracker for {ocp-product-title} and {dedicated-product-title}
-[cols="3,1,1,1,1,1",options="header"]
+[cols="3,1,1,1,1,1,1",options="header"]
 |====
-|Feature |1.21|1.22 to 1.26|1.27|1.28|1.29
+|Feature |1.21|1.22 to 1.26|1.27|1.28|1.29|1.30
 
-|`KafkaBinding` API
+|`NamespacedKafka` annotation
+|-
+|-
+|-
+|-
+|-
+|Deprecated
+
+|{pipelines-shortname} 1.10 and 1.11
+|-
+|-
+|-
+|-
+|-
+|Deprecated
+
+|`enable-secret-informer-filtering` annotation
+|-
+|-
+|-
+|Deprecated
+|Deprecated
+|Deprecated
+
+|Serving and Eventing `v1alpha1` API
+|-
+|-
+|Deprecated
```

Comment:
Interesting... serverless 1.29,1.30 has not removed `v1alpha1` on Red Hat OpenShift Service on AWS even though  it was removed on OpenShift Container Platform and OpenShift Dedicated?
It is not a mistake?

Reply:
@nak3 Thanks Kenjiro. It was just a broken table, now the status for all OpenShift variants is the same.